pkgs/development/tools/txtpbfmt/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "txtpbfmt";
+  version = "unstable-2022-06-08";
+  rev = "fc78c767cd6a4e6e3953f5d72f1e0e4c5811990b";
+
+  src = fetchFromGitHub {
+    owner = "protocolbuffers";
+    repo = "txtpbfmt";
+    inherit rev;
+    sha256 = "sha256-5Pj2REFrzWCzrqdplNlyfX+sJqPjXEld6MFNy0S3MFM=";
+  };
```

Comment:
`rev` can be put inside `fetchFromGitHub` instead of `mkDerivation` as the build process doesn't requires it.

Reply:
Done